specification/maps/data-plane/Route/preview/1.0/route.json

```diff
@@ -828,7 +828,7 @@
     "RouteInstructionsType": {
       "name": "instructionsType",
       "in": "query",
-      "description": "If specified, guidance instructions will be returned. Note that the instructionsType parameter cannot be used in conjunction with routeRepresentation=none",
+      "description": "If specified, guidance instructions will be returned. Note that the instructionsType parameter cannot be used in conjunction with routeRepresentation=none. Includes only the optimized waypoint indices but does not include the route geometry in the response. This parameter value can only be used in conjunction with computeBestOrder=true.",
```

Comment:
@stevemunk The description needs to be updated for parameter "routePresentation" enum value "none"
https://learn.microsoft.com/en-us/rest/api/maps/route/get-route-directions?view=rest-maps-2024-04-01&tabs=HTTP#routerepresentationforbestorder
![image](https://github.com/Azure/azure-rest-api-specs/assets/124096851/fe1def32-f7b2-4eda-abda-212f605308bb)